StackOverflow/Model/QuestionsData.h

```diff
@@ -0,0 +1,18 @@
+//
+//  QuestionsData.h
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface QuestionsData : NSObject
+
+@property (strong, nonatomic) NSString *title;
+@property (strong, nonatomic) NSArray *tags;
+@property (strong, nonatomic) NSString *answer_count;
```

Comment:
We do NOT use _ ( or snake case ) in OC or Swift. We use camel case. i.e. answerCount.